reversing the way we add layers in treestore

weskamm opened the pull request:
Ok, this PR needs a little explanation:
I reversed the order the layers get added in the treestore.
The background is the following: The ol3-layer-collection contains the layers in the order they have been added.
If you add a layer at runtime to the map, it will get appended to the end of the collection / array.
In order to show this newly added layer on top in a layertree, you will need to reverse the collection.
I solved this by using 'Ext.each' with the 'reverse' flag set to true, so the code does the same as before but just in the reversed order.
This way the layertree shows the first / oldest added layers at the bottom, the newly or later added laters get shown on top.

Have a look at the layertree example to see the current problem:
http://rawgit.com/geoext/geoext3/master/examples/tree/panel.html

The layer at the bottom gets shown on top of the map.
I suspect this is exactly the wrong behaviour and this fix addresses this issue.

KaiVolland commented:
I think the most users will expect the layer order of the map to be the same as in the tree. All checks are green so i am :+1:  for merging. Maybe a third person should have a look at this, cause it maybe has impact on several other components.

chrismayer commented:
Hi @weskamm and @KaiVolland,

I agree that most users expect that the order in the tree corresponds to the map's layer order. Somehow I remember some applications where an inverse layer order should be displayed in a tree (for whatever reason). So what do you think about making this configurable?  Maybe by a config option ``inverseLayerOrder`` with the default ``false``? What do you think?

weskamm commented:
You are right @chrismayer, i added a property 'inverseLayerOrder', but the default value is true as that is what is needed for the described behaviour. 
i added a little explanation to the property

KaiVolland commented:
:+1:  :top:

chrismayer commented:
Cool @weskamm :+1: 
BTW, I also meant that the default is the behavior described by you above. Maybe ``false`` was a bit misleading. So all good, please merge if you want.
